### DISCUSSION TITLE
Device not recognized properly

### DISCUSSION THREAD

**Discussion by fifise77** [zwave-js/zwave-js#4264] [General] [answered]
Hello,
First sorry for my broken english

I am trying to associate IDINIO Z-Wave LED Cord Dimmer  (https://www.shada.nl/products/5448/z-wave-led-cord-dimmer) in zwavejs2mqtt but it's reconnized as a ECO-DIM Z-Wave Led dimmer.

The IDINIO Z-Wave LED doesn't react to command send from zwavejs2mqtt most of the time.
 When this happens,  I turn the light on/off manualy on the dimmer, it starts to react again to zwavejs2mqtt for some time.

How can I make  zwavejs2mqtt recognize the device properly and fix this device not reacting from  zwavejs2mqtt ?

thanks in advance for your help

**Comment by robertsLando** [comment]
@blhoward2

**Comment by blhoward2** [comment, marked as answer]
Please open an issue under the node-zwave-js project. We’re going to need debug level logs of you trying to operate it to see what’s going on. We can fix the label but that has zero to do with it working or not.
